Edited the Documentation for the changes to the LLAPI (#3733)
Unity-Technologies/ml-agents: docs/Python-API.md

- **UnityEnvironment** — the main interface between the Unity application and
  your code. Use UnityEnvironment to start and control a simulation or training
  session.
- **BehaviorName** - is a string that identifies a behavior in the simulation.
- **AgentId** - is an `int` that serves as unique identifier for Agents in the
  simulation.
- **DecisionSteps** — contains the data from Agents belonging to the same
  "Behavior" in the simulation, such as observations and rewards. Only Agents
  that requested a decision since the last call to `env.step()` are in the
  DecisionSteps object.
- **TerminalSteps** — contains the data from Agents belonging to the same

>   "Behavior" in the simulation, such as observations and rewards. Only Agents

Reviewer: Only agents whose episodes ended since...

>   "Behavior" in the simulation, such as observations and rewards. Only Agents
>   that has their episode end since the last call to `env.step()` are in the

Author: ```suggestion
  "Behavior" in the simulation, such as observations and rewards. Only Agents
  whose episode ended since the last call to `env.step()` are in the
```

   move forward until an Agent in the simulation needs a input from Python to act.
 - **Close : `env.close()`** Sends a shutdown signal to the environment and terminates
   the communication.
 - **Get Behavior Names : `env.get_behavior_names()`** Returns a list of `BehaviorName`.
   Note that the number of groups can change over time in the simulation if new
   Agent behaviors are created in the simulation.
 - **Get Behavior Spec : `env.get_behavior_spec(behavior_name: str)`** Returns
   the `BehaviorSpec` corresponding to the behavior_name given as input. A
   `BehaviorSpec` contains information such as the observation shapes, the action
   type (multi-discrete or continuous) and the action shape. Note that the `BehaviorSpec`
   for a specific group is fixed throughout the simulation.
 - **Get Steps : `env.get_steps(behavior_name: str)`**
   Returns a tuple `DecisionSteps, TerminalSteps` corresponding to the behavior_name
   given as input.
   The `DecisionSteps` contains information about the state of the agents
   **that need an action this step** and have the behavior behavior_name.
   The `TerminalSteps` contains information about the state of the agents

>    **that had their episode end** and have the behavior behavior_name.

Author: ```suggestion
   **whose episode ended** and have the behavior behavior_name.
```
